1/backend/src/resolvers.js

```diff
@@ -0,0 +1,86 @@
+const jwt = require('jsonwebtoken');
+const fs = require('fs');
+
+data = require('./data');
+
+let SECRET_KEY = fs.readFileSync('./src/key/secret.key', 'utf8');
+
+const resolvers = {
+    Query: {
+        todos: () => JSON.parse(JSON.stringify(data.todos))
+    },
+    Mutation: {
+        addTodo: (parent, args, context) => {
+            if(args.message != "" && args.message != null && context.token) {
+                let newTodo = {
+                    id: findNextId(),
+                    message: args.message,
+                    completed: false,
```

Comment:
You can also do:
```js
let newTodo = {
  ...args,
  id: findNextId(),
  completed: false,
}
```
which assigns all of `args` to the new todo and overrides it with your initial values.